apps/btshell/src/cmd.c

```diff
@@ -4235,7 +4235,7 @@ static const struct shell_cmd_help sync_stats_help = {
 #endif
 #endif
 
-#if MYNEWT_VAL(BLE_ISO_BROADCAST_SOURCE)
+#if MYNEWT_VAL(BLE_AUDIO) && MYNEWT_VAL(BLE_ISO_BROADCAST_SOURCE)
```

Comment:
We should have a dedicated syscfg option like `BLE_AUDIO_BROADCAST_SOURCE` actually IMO